packages/e2e/cypress/e2e/app/savedDashboards.cy.ts

```diff
@@ -37,9 +37,7 @@ describe('Dashboard List', () => {
         cy.contains('tr', 'Untitled dashboard').find('button').click();
         // click on rename
         cy.findByRole('menuitem', { name: 'Rename' }).click();
-        cy.findByLabelText('Enter a memorable name for your dashboard *')
-            .clear()
-            .type('e2e dashboard');
+        cy.findByLabelText('Name *').clear().type('e2e dashboard');
```

Comment:
input label doesn't need to be "enter..." when the user is editing a dashboard - feels unnecessary